.github/workflows/frontend_tests.yml

```
@@ -28,3 +28,9 @@ jobs:
         run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
       - name: Run frontend tests
         run: PYTHONIOENCODING=utf-8 python -m scripts.run_frontend_tests --run_minified_tests --skip_install --check_coverage
+      - name: Report failure if failed on oppia/oppia develop branch
+        if: ${{ github.event_name == 'push' && github.repository == 'oppia/oppia' && github.ref == 'refs/heads/develop'}}
+        uses: ./.github/actions/send-webhook-notification
+        with:
+          message: "Frontend test failed on develop branch!"
+          webhook-url: ${{ secrets.BUILD_FAILURE_ROOM_WEBHOOK_URL}}
```

Maybe also leave a comment specifying which room this is for? That will help anyone who wants to receive these notifications

```suggestion
          webhook-url: ${{ secrets.BUILD_FAILURE_ROOM_WEBHOOK_URL }}
```

---

I'm avoiding adding any more info about the room here as it will only be handled by the dev workflow team! Also, the URL is expected to change as required and the name of the variable explains the actual usage of the room (BUILD_FAILURE_ROOM).

I'm also fine with adding more info if you think that's important?

---

Done! (space addition)